src/locale/keymap-util.c

```diff
@@ -780,3 +783,212 @@ int x11_convert_to_vconsole(Context *c) {
 
         return modified;
 }
+
+bool locale_gen_check_available(void) {
+#if HAVE_LOCALEGEN
+        if (access(LOCALEGEN_PATH, X_OK) < 0) {
+                if (errno != ENOENT)
+                        log_warning_errno(errno, "Unable to determine whether " LOCALEGEN_PATH " exists and is executable: %m");
```

Comment:
maybe add `…, assuming it is not: …`

i.e. make the log message say what happened but *also* say what our reaction is, in particular if we don't actually fail but proceed.